html/index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html style="font-size: 10px;">
```

Comment:
html 태그에 작성된 인라인 스타일은 css 파일로 분리하는 것이 나중에 헷갈리지 않고 좋을 것 같아요.

추가적으로 html에 [언어속성](https://developer.mozilla.org/ko/docs/Web/HTML/Global_attributes/lang)이 누락되었네요

```suggestion
<html lang="ko">
```